exporters/tokens/src/generators/mixinGenerator.ts

```diff
@@ -0,0 +1,23 @@
+import { Token, TokenGroup } from '@supernovaio/sdk-exporters';
+import { sortTokens, tokenVariableName } from '../helpers/tokenHelper';
+
+export const generateMixinFromTokens = (
+  tokens: Token[],
+  mappedTokens: Map<string, Token>,
+  tokenGroups: Array<TokenGroup>,
+  tokenPrefix: string,
+  group: string,
+  hasParentPrefix: boolean,
+  sortByNumValue: boolean,
+): string => {
+  const sortedTokens = sortTokens(tokens, tokenGroups, hasParentPrefix, group, sortByNumValue);
+  const variables = sortedTokens
+    .map((token) => {
+      const name = tokenVariableName(token, tokenGroups, hasParentPrefix);
+
+      return `    --${tokenPrefix}color-${name}: #{${name}};`;
```

Comment:
**suggestion (blocking):** I think we have a constant for the empty space somewhere. We should use it instead of this hardcoded value.

Reply:
I actually had it, but then I mistakenly deleted it during refactoring